cpg_workflows/workflow.py

```diff
@@ -929,7 +929,7 @@ def __init__(
                 'Workflow already initialised. Use get_workflow() to get the instance'
             )
 
-        self.dry_run = dry_run or get_config()['workflow'].get('dry_run')
+        self.dry_run = dry_run or get_config(True)['workflow'].get('dry_run')
```

Comment:
Since this is the only change where true is passed to get_config, does that mean that we'd only print the config during a dry-run?

Reply:
Ah sorry, I mis-read. Ignore my above comment.

Reply:
As far as I can track back this is the first instance of get_config being called in the workflow startup process. I'm not sure if we'd want to put a True in the next one as well, in case we run with `dry_run==True`, which would make the first get_config() call the one after this.